event-service-rest/src/main/java/com/restapi/rest/EventApi.java

```diff
@@ -0,0 +1,33 @@
+package com.restapi.rest;
+
+import com.restapi.dto.Event;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EventApi {
```

Comment:
why are you using a different object for API response?

Reply:
> why are you using a different object for API response?

explained